### DISCUSSION TITLE
Can you use vercel for free with a GitHub org?

### DISCUSSION THREAD

**Discussion by Andre601** [vercel/community#1708] [Help] [answered]
### Summary

I would like to use Vercel to allow PR previews for sites I host.
Unfortunately, those repositories are part of a GitHub Organization, meaning I have to make a Team and therefore use a Pro plan here, which I can't due to financial struggles right now.

Is there any way I can use Vercel while still having my repositories as part of an Org?

**Comment by swarnava** [comment, marked as answer]
Unfortunately no. GitHub organization requires Pro plan in order to deploy project. You can request for Sponsorship for Open Source project: https://vercel.com/guides/can-vercel-sponsor-my-open-source-project
